File: Python/european_parrot.py
from parrot import Parrot, ParrotType


class EuropeanParrot(Parrot):

    def __init__(self, number_of_coconuts, voltage, nailed):
        super().__init__(number_of_coconuts, voltage, nailed)

    def speed(self):
        return self._base_speed()

    def cry(self):
        return "Sqoork!"

File: Java/src/main/com/parrot/EuropeanParrot.java
package com.parrot;

public class EuropeanParrot  extends Parrot {
    public EuropeanParrot( int numberOfCoconuts, double voltage, boolean isNailed) {
        super(numberOfCoconuts, voltage, isNailed);
    }
    @Override
    public double getSpeed() {
        return BASE_SPEED;
    }

    @Override
    protected String getCry() {
        return "Sqoork!";
    }
}
